src/layouts/partials.tsx

```diff
@@ -249,3 +253,27 @@ export function SubNavigation(props: ISubNavigationProperties): ReactElement {
     </div>
   );
 }
+
+export function CommandLineAlternative(props: ICommandLineAlternativeProperties): ReactElement {
+  return (
+    <>
+      <h4 className="govuk-heading-s">On the commandline</h4>
+
+      <p>
```

Comment:
```
      <p>
        You can also view the same information on the commandline, to see
        details {optional property for additional context} use:
      </p>
```

To allow us to customise the error a bit more?

Reply:
Good shout. Amended.